libs/math/include/math/TQuatHelpers.h

```diff
@@ -244,6 +244,10 @@ class TQuatFunctions {
 
     friend inline
     QUATERNION<T> MATH_PURE slerp(const QUATERNION<T>& p, const QUATERNION<T>& q, T t) {
+        if (p == q) {
```

Comment:
This is unrelated to the rest of this PR

Reply:
Deleted